### PR TITLE
Update GPT3-fine-tune-API-metadata-extraction notebook

### PR DESCRIPTION
**Pull request by juhoinkinen**
OpenAI client has changed since running this notebook (v1.0 was released 6.11.2023, and now the latest is already v1.3.7).

Also the babbage model [has been deprecated and will shutdown 4.1.2024](https://platform.openai.com/docs/deprecations/base-gpt-models), so I updated to use babbage-002 base model as recommended.

For my run the results got worse (overall similarity 0.7886 -> 0.7216) but at this point I think it does not matter so much.

BTW, at openai.com one can now see logs and training loss of the fine-tune run: 
image

### CODE REVIEW

**Review by osma — APPROVED** [review]
Looks good, this is a nice step forward
